docs/Configuration.md

```diff
@@ -190,10 +193,40 @@ trigger:
    limit: 10 mb
 ```
 
+For `time`, the `limit` field is a string which defines the time to roll the 
+file. The limit field supports the following units(second will be used if the
+unit is not specified), case does not matter:
+
+- second[s]
+- minute[s]
+- hour[s]
+- day[s]
+- week[s]
+- month[s]
+- year[s]
+
+> Note: The log file will be rolled at the integer time. For example, if the
+`limit` is set to `2 day`, the log file will be rolled at 0:00 every other a
+day, regardless of the time `log4rs` was started or the log file was created.
+This means that the initial log file will be likely rolled before the limit
+is reached.
+
+> Note: Setting "roller.is_pre_process"(In the next section, we will introduce)
+to true can ensure that the logs are rolled in the correct position instead of
+leaving a single line of logs in the previous log file.
```

Comment:
```suggestion
> Note: Setting "roller.is_pre_process"(In the next section, we will introduce)
> to true can ensure that the logs are rolled in the correct position instead of
> leaving a single line of logs in the previous log file.
```